config/goGenerate.coffee

```diff
@@ -0,0 +1,28 @@
+# goGenerate module generates a JSON object which will be parsed by `go generate`
+# tool in order to create static configuration for Go programs. One can put Go's
+# template actions into provided strings. Values available:
+#
+#  {{.Group}} - will be replaced by either `production` or `development` string
+#               depending on build environment and provided environment.
+#
+#  {{.Environment}} - will be replaced by `production`, `managed`, `development`
+#                     or `devmanaged` string depending on provided environment.
+#
+module.exports = (options) ->
+  environment    : options.environment
+  buckets        :
+    publicLogs   :
+      name       : options.publicLogsS3BucketName
+      region     : 'us-east-1'
+  endpoints      :
+    ip           : "https://#{options.proxySubdomain}.koding.com/-/ip"
+    ipCheck      : "https://#{options.proxySubdomain}.koding.com/-/ipcheck"
+    kdLatest     : "https://koding-kd.s3.amazonaws.com/{{.Group}}/latest-version.txt"
```

Comment:
this can be hard to achieve for environments like `sandbox` which gets their version from `development` bucket. Maybe it's better to just create `sandbox` bucket on S3 so we can indeed get rid of {{.Group}}.